words/teams.txt

```diff
@@ -1,10 +1,27 @@
 alliance
+assembly
 band
+batch
+bunch
+club
+cluster
```

Comment:
"This is a cluster" to me implies "this is a clusterf***" so I'd be more hesitant to add it.